internal/data/postgres/conversations.go

```diff
@@ -63,14 +62,17 @@ func (r ConversationsQ) Select() ([]data.Conversation, error) {
 	return result, errors.Wrap(err, "failed to select conversations")
 }
 
-func (r ConversationsQ) Upsert(conversation data.Conversation) error {
+func (r ConversationsQ) Upsert(conversations ...data.Conversation) error {
 	updateStmt, args := sq.Update(" ").
-		Set("title", conversation.Title).
-		Set("members_amount", conversation.MembersAmount).
+		Set("title", sq.Expr("EXCLUDED.title")).
+		Set("members_amount", sq.Expr("EXCLUDED.members_amount")).
 		MustSql()
 
-	query := sq.Insert(conversationsTableName).SetMap(structs.Map(conversation)).
-		Suffix("ON CONFLICT (id) DO "+updateStmt, args...)
+	query := sq.Insert(conversationsTableName).Columns("id", "title", "members_amount")
```

Comment:
It will be better to move raw strings to constants in order to reuse in future if any filters appear and in case of updating column name you may update just one constant instead of searching everywhere this column

Reply:
Also if you wish, you can create some array with columns and pass like `.Columns(conversationsColumns...)`, but it's up to you. Again if we have more columns in future it will be more convenient.